recipes/dbus/1.x.x/conanfile.py

```diff
@@ -104,7 +104,9 @@ def generate(self):
             tc.project_options["checks"] = False
             tc.project_options["doxygen_docs"] = "disabled"
             tc.project_options["modular_tests"] = "disabled"
-            tc.project_options["session_socket_dir"] = str(self.options.session_socket_dir)
+            tc.project_options["system_socket"] = self.options.get_safe("system_socket", "")
+            tc.project_options["system_pid_file"] = self.options.get_safe("system_pid_file", "")
+            tc.project_options["session_socket_dir"] = self.options.get_safe("session_socket_dir", "")
```

Comment:
@Knapock Just wrap these with `str`.

```suggestion
            tc.project_options["system_socket"] = str(self.options.get_safe("system_socket", ""))
            tc.project_options["system_pid_file"] = str(self.options.get_safe("system_pid_file", ""))
            tc.project_options["session_socket_dir"] = str(self.options.get_safe("session_socket_dir", ""))
```

Reply:
done